package.json

```diff
@@ -55,9 +56,9 @@
     "template"
   ],
   "scripts": {
-    "test": "NODE_ENV=test mocha --reporter spec --timeout 30s --bail test/**/*.js",
-    "test-cov": "NODE_ENV=test istanbul cover _mocha -- --reporter dot --timeout 2m --bail test/**/*.js",
-    "test-travis": "NODE_ENV=test istanbul cover _mocha --report lcovonly -- --reporter dot --timeout 2m --bail test/**/*.js",
+    "test": "jest --forceExit",
```

Comment:
I'm not a fan of this `--forceExit` [thing](https://facebook.github.io/jest/docs/en/cli.html#forceexit) here, but CLI wasn't exiting after passing tests. This happens even if you chose to run _any_ single test file (meaning it's not one particular test's fault).  

I believe this is due to `supertest` not closing connections properly. [Koa itself](https://github.com/koajs/koa/blob/master/package.json#L7) is using the same approach. There has also been [some discussion](https://github.com/facebook/jest/issues/1456#issuecomment-246419649) in the past to work around this issue over at the Jest repo.